test/hotspot/jtreg/vmTestbase/nsk/jvmti/unit/functions/rawmonitor/rawmonitor.cpp

```diff
@@ -1,4 +1,5 @@
 /*
+s
```

Comment:
this feels like an accidental `s` added to the comment but it's the same in openjdk/jdk https://github.com/openjdk/jdk/blob/4ec1ae109710aa150e27acf5706475d335c4655c/test/hotspot/jtreg/vmTestbase/nsk/jvmti/unit/functions/rawmonitor/rawmonitor.cpp#L1-L5 so needs fixing there

Reply:
Thanks. Created https://github.com/openjdk/jdk/pull/20051

Reply:
Good catch and thanks @jerboaa for fixing in trunk. I'll open a backport to 21u.